internal/testdata/channels.go

```diff
@@ -100,45 +92,6 @@ func (l LedgerNetwork) GetLedgerLookup(seeker types.Address) virtualfund.GetTwoP
 	}
 }
 
-// createLedgerNetwork returns active, funded consensus_channels connecting the supplied
-// actors according the the supplied edge list.
-//
-// Edges specify actors via their indices in the actors slice,
-// and each edge is ordered [leader, follower].
-func createLedgerNetwork(actors []testactors.Actor, edges [][2]int) LedgerNetwork {
```

Comment:
This change seems unrelated: are you removing because it is currently unused?

If so, just looping in @NiloCK who I think put some effort into this.